### PR TITLE
Fix custom presentation for PhET

### PR DESCRIPTION
**Pull request by erikos**
Add custom icon and follow the design for the presentation.

follow up for #637

### CODE REVIEW

File: kolibri_explore_plugin/assets/src/customApps.js [hunk 1]
```diff
@@ -295,6 +295,7 @@ export const ChannelCardIcons = {
   '08897e003ea9489eb3d86fc94ba08c21': 'support-ukraine.png',
   '608fd926be60462aba2feccba6c050c9': 'harper-collins.png',
   '6da97a60cbd44e918137827bd5425efb': 'code-org.png',
+  '8fa678af1dd05329bf3218c549b84996': 'phet-interactive-simulations.png', // phet spanish
```
Comment (jprvita) [inline kolibri_explore_plugin/assets/src/customApps.js:298]: Don't we want the same customization for the Part English as well?

**Review by jprvita — APPROVED** [review]
One small question about the consistency between the English and Spanish versions of Phet, but LGTM otherwise.

File: kolibri_explore_plugin/assets/src/customApps.js (hunk 1 above)
Comment (erikos) [reply kolibri_explore_plugin/assets/src/customApps.js:298]: Yes, for some reason I could not figure out the English channel has the right channel icon. I could make it explicit, I guess?

Comment (jprvita) [reply kolibri_explore_plugin/assets/src/customApps.js:298]: Ok, IIUC we do NOT need the same customization for Phet English. In that case, this LGTM.

**Review by jprvita — APPROVED** [review]

**Pull request merged**